static/src/javascripts/projects/common/modules/video/video-container.js

```diff
@@ -67,6 +76,47 @@ define([
         });
     }
 
+    // iniLazyLoadImages is used for when a user scrolls i.e. thin viewport
+    // and fetchImage is used for when the carousel controls are used i.e. fat viewport
+    function initLazyLoadImages() {
```

Comment:
If I could find a way to do `element.on('transform')` we wouldn't have to have the two different ways of doing this.

Or we could have a way of [`element.scrollIntoView({ behaviour: 'smooth' })`](https://developer.mozilla.org/en/docs/Web/API/Element/scrollIntoView#Parameters). 

Writing either of those seemed harder than this.